cpp/include/cudf/functions.h

```diff
@@ -381,19 +381,23 @@ gdf_error gdf_hash_partition(int num_input_cols,
 /* prefixsum */
 
 /* --------------------------------------------------------------------------*/
-/** 
- * @brief  Computes the prefix sum of a column
- * 
- * @param inp Input column for prefix sum with null_count = 0
- * @param out The output column containing the prefix sum of the input
- * @param inclusive Flag for applying an inclusive prefix sum
- * 
- * @returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
+/**
+ * @brief  Computes the prefix sum of a column.
+ * The Null values are skipped for the operation,
+ * and the output column has same null values with the input column.
+ *
+ * @param[in] input The input column for prefix sum
+ * @param[out] output The pre-allocated output column containing
+ * the prefix sum of the input
+ * @param[in] inclusive The flag for applying an inclusive prefix sum if true,
+ * an exclusive prefix sum if false.
+ *
+ * @returns   GDF_SUCCESS if the operation was successful,
+ * otherwise an appropriate error code.
+ * GDF_UNSUPPORTED_DTYPE if input->dtype is not an arithmetic type.
  */
 /* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_generic(gdf_column *inp, gdf_column *out, int inclusive);
+gdf_error gdf_prefixsum_generic(gdf_column *inp, gdf_column *out, bool inclusive);
```

Comment:
I believe this is an errant leftover from a merge as its the old API that will be replace by your new PR.